homeassistant/components/wink/water_heater.py

```diff
@@ -103,9 +103,8 @@ def operation_list(self):
                 op_list.append(ha_mode)
             else:
                 error = (
-                    "Invalid operation mode mapping. "
-                    + mode
-                    + " doesn't map. Please report this."
+                    f"Invalid operation mode mapping. "
```

Comment:
There are no variables in this string...
```suggestion
                    "Invalid operation mode mapping. "
```